calculator/operations.py

```diff
@@ -1,23 +1,28 @@
-
 def add(*args):
-    # your implementation here
-    pass
+    return sum(args)
 
 def subtract(*args):
-    # your implementation here
-    pass
+    total = args[0]
+    for number in args[1:]:
+       total -= number
+       
+    return total
 
 def multiply(*args):
-    # your implementation here
-    pass
+    total = 1
+    for number in args:
+        total *= number
+    
+    return total
 
 def divide(*args):
-    # your implementation here
-    pass
+    total = float(args[0])
```

Comment:
discovery of the day/night/morning
```In Python 3, / is float division
In Python 2, / is integer division (assuming int inputs)```

@jsymons and I tried `from __future__ import division` in `main.py` but Python 2.7 was still doing integer division so we decided to cast the first argument to a float. admittedly not an ideal solution - feedback would be awesome.